SocialFilter/sections/general_elemhide.txt

```diff
@@ -568,6 +568,8 @@ designboom.com,livescience.com,pbtech.co.nz,peoplesworld.org,space.com##.fa-face
 ##.share-on-facebook
 ##.share-on-twitter
 ~grammarly.com,~youtube.com##.share-panel
+! hide share button in youtube player
+##.ytp-share-button
```

Comment:
Why generic elemhide?
youtube.com##.ytp-share-button

I need this rule too, but did not find good selectors for it :(
stopgame.ru##pjsdiv[id^="oframeplayer_"] pjsdiv[style*="px; top: 31px; opacity: 1;"]

Reply:
Because this rule `youtube.com##.ytp-share-button` did not work in Safari.